plasma_framework/contracts/mocks/framework/utils/PriorityQueueTest.sol

```diff
@@ -0,0 +1,78 @@
+pragma solidity ^0.5.0;
+import "../../../src/framework/utils/PriorityQueue.sol";
+
+/**
+ * @title PriorityQueue
+ * @dev Min-heap priority queue implementation.
+ */
+contract PriorityQueueTest{
```

Comment:
Unfortunately it is necessary, since in the web3 model I cannot get the return value from a transaction, that Is why I prefer to emit events there.

I will move it to another directory, though.